packages/embed-grid/index.html

```diff
@@ -1,6 +1,7 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
+    <base href="#BASE_URL#" />
```

Comment:
You can configure the base to be something else though, so possible to still build with an env var set and have the base tag be set.
We use `/ide/` in dev mode, for example.

Reply:
Ok makes sense. Also as you said it makes a Jetty rewrite easier since Jetty can use sed or something to just replace this tag instead of having to find where to insert it